src/NetAppFiles/NetAppFiles/Volume/NewNetAppFilesVolume.cs

```diff
@@ -249,12 +249,30 @@ public class NewAzureRmNetAppFilesVolume : AzureNetAppFilesCmdletBase
         [PSArgumentCompleter("Basic", "Standard")]
         public string NetworkFeature { get; set; }
 
+        [Parameter(            
+            Mandatory = false,
+            HelpMessage = "Pool Resource Id used in case of creating a volume through volume group.")]
+        [ValidateNotNullOrEmpty]
+        public string CapacityPoolResourceId { get; set; }
+
         [Parameter(
-            ParameterSetName = FieldsParameterSet,
             Mandatory = false,
-            HelpMessage = "Network Sibling Set ID (GUID) for the group of volumes sharing networking resources example (9760acf5-4638-11e7-9bdb-020073ca3333).")]
+            HelpMessage = "Proximity placement group associated with the volume.")]
         [ValidateNotNullOrEmpty]
-        public string NetworkSiblingSetId { get; set; }
+        public string ProximityPlacementGroup { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Volume spec name is the application specific designation or identifier for the particular volume in a volume group for e.g. data, log.")]
+        [ValidateNotNullOrEmpty]
+        public string VolumeSpecName { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Application specific placement rules for the particular volume.")]
+        [ValidateNotNullOrEmpty]
+        public IList<PSKeyValuePairs> PlacementRules { get; set; }
```

Comment:
As PowerShell suggest use singularity for parameters. You can find doc [here](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/parameter-best-practices.md#singularity). Please rename `PlacementRules` to `PlacementRule`.